database_scheme.sql

```diff
@@ -1,3 +1,5 @@
+CREATE TYPE class AS ENUM ('activity', 'emotion');
```

Comment:
I don't like the name of the type... It should be a bit more descriptive to my mind

Reply:
Yes, I thought about it, too. But if we'll make it more informative it'll be much longer. What about naming this field `activity_or_emotion`?

Reply:
I'm not sure if it's perfect but much better